openpype/hosts/nuke/plugins/publish/extract_slate_frame.py

```diff
@@ -15,159 +20,194 @@ class ExtractSlateFrame(openpype.api.Extractor):
+        self.add_comment_slate_node(instance, slate_node)
+
+        # solve output name if any is set
+        _output_name = output_name or ""
+        if _output_name:
+            _output_name = "_" + _output_name
+
+        slate_first_frame = first_frame - 1
 
-        temporary_nodes = []
         collection = instance.data.get("collection", None)
 
         if collection:
             # get path
             fname = os.path.basename(collection.format(
                 "{head}{padding}{tail}"))
             fhead = collection.format("{head}")
-
-            collected_frames_len = int(len(collection.indexes))
-
-            # get first and last frame
-            first_frame = min(collection.indexes) - 1
-            self.log.info('frame_length: {}'.format(frame_length))
-            self.log.info(
-                'len(collection.indexes): {}'.format(collected_frames_len)
-            )
-            if ("slate" in instance.data["families"]) \
-                    and (frame_length != collected_frames_len):
-                first_frame += 1
-
-            last_frame = first_frame
         else:
-            fname = os.path.basename(instance.data.get("path", None))
+            fname = os.path.basename(fpath)
             fhead = os.path.splitext(fname)[0] + "."
-            first_frame = instance.data.get("frameStartHandle", None) - 1
-            last_frame = first_frame
 
         if "#" in fhead:
             fhead = fhead.replace("#", "")[:-1]
 
-        previous_node = node
-
-        # get input process and connect it to baking
-        ipn = self.get_view_process_node()
-        if ipn is not None:
-            ipn.setInput(0, previous_node)
-            previous_node = ipn
-            temporary_nodes.append(ipn)
-
-        if not self.viewer_lut_raw:
+        self.log.debug("__ first_frame: {}".format(first_frame))
+        self.log.debug("__ slate_first_frame: {}".format(slate_first_frame))
+
+        # Read node
+        r_node = nuke.createNode("Read")
+        r_node["file"].setValue(fpath)
+        r_node["first"].setValue(first_frame)
+        r_node["origfirst"].setValue(first_frame)
+        r_node["last"].setValue(last_frame)
+        r_node["origlast"].setValue(last_frame)
+        r_node["colorspace"].setValue(instance.data["colorspace"])
+        previous_node = r_node
+        temporary_nodes = [previous_node]
+
+        # only create colorspace baking if toggled on
+        if bake_viewer_process:
+            if bake_viewer_input_process:
+                # get input process and connect it to baking
+                ipn = get_view_process_node()
+                if ipn is not None:
+                    ipn.setInput(0, previous_node)
+                    previous_node = ipn
+                    temporary_nodes.append(ipn)
+
+            # add duplicate slate node and connect to previous
+            duply_slate_node = duplicate_node(slate_node)
+            duply_slate_node.setInput(0, previous_node)
```

Comment:
Crashes here. The duplicated slate node is not created, so no way to wire it:
```

Instance: 
renderCompMain
Message: 
'NoneType' object has no attribute 'setInput'
Line: 
154
Traceback: 
Traceback (most recent call last):
  File "C:\Users\robur\OpenPype\.venv\lib\site-packages\pyblish\plugin.py", line 522, in __explicit_process
    runner(*args)
  File "C:\Users\robur\OpenPype\openpype\hosts\nuke\plugins\publish\extract_slate_frame.py", line 56, in process
  File "C:\Users\robur\OpenPype\openpype\hosts\nuke\plugins\publish\extract_slate_frame.py", line 154, in render_slate
AttributeError: 'NoneType' object has no attribute 'setInput'
```

Reply:
Maybe it can be something with system clipboard breaking the nuke node? Some locale?
Is it possible to use Qt clipboard maybe?